sqlair.go

```diff
@@ -24,6 +24,10 @@ import (
 //	err := q.Get(resultMap) // => sqlair.M{"name": "Fred", "postcode": 10031}
 type M map[string]any
 
+// S is a slice type that, as with other named slice types, can be used with
+// SQLair to pass a slice of input values.
+type S []any
```

Comment:
The `S` type is in the slice support spec. I think it adds convenience at least in that the user does not have to design a custom type, since using `[]any` is not possible.